data/zilliqa/assets/zil14d6wwelssqumu6w9c6kaucz2r57z34cxuh96lf/meta.json

```diff
@@ -0,0 +1,16 @@
+{
+	"name": "Shards",
+	"symbol": "SHARDS",
+	"web": "https://shards.robounicats.com/",
+	"whitepaper": "https://shards.robounicats.com/data/shards_whitepaper_0.1.pdf",
+	"publicTeam": true,
+	"product": true,
```

Comment:
remove this, will be good to add once you have enough holders/activity/product usage

Reply:
Ok